app/javascript/mastodon/actions/accounts_typed.ts

```diff
@@ -0,0 +1,97 @@
+import { createAction } from '@reduxjs/toolkit';
+
+import type { ApiAccountJSON } from 'mastodon/api_types/accounts';
+import type { ApiRelationshipJSON } from 'mastodon/api_types/relationships';
+
+export const revealAccount = createAction<{
+  id: string;
+}>('accounts/revealAccount');
+
+export const importAccounts = createAction<{ accounts: ApiAccountJSON[] }>(
+  'accounts/importAccounts',
+);
```

Comment:
I'm noticing that `importAccounts` used to work on normalized (internal) objects but was changed to do the normalization later in the process (as part of the reducer). What is the rationale behind this change?